resources/views/partials/top_nav.blade.php

```diff
@@ -1,3 +1,8 @@
+@php
+    use App\Models\Donation;
+    use Carbon\Carbon;
+@endphp
+
```

Comment:
Use `@use()` instead of `@php`.

Also use `Illuminate\Support\Carbon' instead of `Carbon\Carbon`: https://github.com/HDInnovations/UNIT3D-Community-Edition/pull/2251